gradle.properties

```diff
@@ -15,14 +15,14 @@ springDependencyManagementPluginVersion=1.0.3.RELEASE
 
 springCloudSleuthVersion=2.0.2.RELEASE
 dockerComposePluginVersion=0.6.6
-eventuateCdcVersion=0.7.0.RELEASE
+eventuateCdcVersion=0.8.0.BUILD-SNAPSHOT
 eventuateCommonVersion=0.10.0.RELEASE
 eventuateMessagingKafkaVersion=0.10.0.RELEASE
 eventuateMessagingActiveMQVersion=0.7.0.RELEASE
-eventuateMessagingRabbitMQVersion=0.8.0.RELEASE
+eventuateMessagingRabbitMQVersion=0.9.0.BUILD-SNAPSHOT
 eventuateMessagingRedisVersion=0.8.0.RELEASE
 
 eventuateCommonImageVersion=0.9.0.RELEASE
-eventuateCdcImageVersion=0.7.0.RELEASE
+eventuateCdcImageVersion=local-test
 
-version=0.26.0-SNAPSHOT
+version=0.27.0-SNAPSHOT
```

Comment:
Why 0.27.0-SNAPSHOT?

Reply:
Because tram updated, I updated it too. I did it wrong?

Reply:
The version=...SNAPSHOT version needs to be increment after a release.
But that has already been done.
You can look at branches to see the releases:

![image](https://user-images.githubusercontent.com/246234/86275992-fcd53900-bb88-11ea-89c3-ccc93ba8dab7.png)

Reply:
Chris, because it anyway requires cdc update, can I fix it after release? And it seems I incremented eventuate-messaging-rabbitmq version correctly.

Reply:
And according to branches cdc I updated correctly too.